src/rpc/blockchain.cpp

```diff
@@ -481,7 +481,7 @@ static std::vector<RPCResult> MempoolEntryDescription() { return {
     RPCResult{RPCResult::Type::BOOL, "unbroadcast", "Whether this transaction is currently unbroadcast (initial broadcast not yet acknowledged by any peers)"},
 };}
 
-static void entryToJSON(const CTxMemPool& pool, UniValue& info, const CTxMemPoolEntry& e) EXCLUSIVE_LOCKS_REQUIRED(pool.cs)
+static void entryToJSON(const CTxMemPool& pool, UniValue& info, const CTxMemPoolEntry& e, bool include_fee_fields) EXCLUSIVE_LOCKS_REQUIRED(pool.cs)
```

Comment:
I agree with this approach of adding a bool similar to address/reqSigs, and think you should add a TODO to remove this in v24

Perhaps naming it `include_old_fee_fields` or something might be better because this could be slightly confusing

Reply:
added the comments and renamed to `include_deprecated_fee_fields` to better associate it with `isDeprecatedRPCEnabled("fees")`